### DISCUSSION TITLE
Any way to capture a stream for longer than 12 hours?

### DISCUSSION THREAD

**Discussion by Zorklis** [aclap-dev/video-downloadhelper#1430] [Help and Q&A]
### Your question

So I'm trying to capture some livestreams and they end up longer than 12 hours, I noticed around the 12 hour mark DownloadHelper stops the capture without starting another capture or anything

### Addon & CoApp version and details

Version 9.1.0.41 (beta), target mozilla, locale en-US.

Companion App installed (v2.0.19)

C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I think the longest livestream I ever captured with VDH was something like 8 hours.  It was, as I recall, a livestream of a round of the ladies' Olympic golf from Japan a few years ago.  But VDH stopped its recording because the broadcast ended, not because of any error & not prematurely.  I don't believe there is any limit within VDH on how long it can record something.  The limits are more likely to be your disk space & any cap your ISP might place on either the duration of a connection or the volume of data you have downloaded.  More likely.  Not definitely.  I'm just guessing here at possibilities.  It's also possible the stream itself did something that looked like it was terminating.  With a stream that long, you are also opening yourself up to the possibility of a line error, a data transmission error, causing a premature termination.  I certainly am not going to bother even trying to let my system be busy downloading something for over 12 hours just as an experiment.  So I think you are largely on your own with this.  If you can give the URL of an archived livestream that anybody can access without having to log in, I would be willing to try downloading that.  Presumably, the serving web site will not throttle the download too badly & something over 12 hours long can be downloaded in something considerably less than 12 hours.  I'd be willing to try that.  But I am not willing to connect to a livestream & let it run for over 12 hours just to see if that works.  So if you have the URL of a free archived livestream, post it here & let's see how it goes.

Also, post your environment information.  Go into the VDH settings, click the `Copy information to clipboard` button, then paste that into a post here.  You should have done that on your original post.  You could edit that post to include that information that currently just shows a spot.  This section of the problem creation dialog is there for a reason.  You should not skip it.  Fill it in whenever you post a problem report.

**Reply by Zorklis** [reply]
Well you're wrong on the disk space, I wouldn't write if that was the case. Nor is it my ISP or wifi lol.

It's not an archived livestream, it's an on going stream, as in live tv. It could certaintly be the website/their player/system having a 12 hour limit but I am leaning towards doubting it, that's why I started this discussion.

**Reply by GrampaWildWilly** [reply]
What is stopping you from just posting the clipboard information?  There's rather important items still missing from your environment information, like your operating system with release number & browser with release number.

I was just guessing at possibilities.  Since this is apparently just a never-ending livestream, I'm not about to try to record it.  But if you always get cut off at 12 hours, that points to limits imposed by external influences.  These could be the serving web site, your ISP, I don't know.  I'm pretty sure VDH doesn't impose that limit, but then again, I'm just another user.  I don't have access to the source code.  And it's doubtful I'd understand it even if I saw it.

It might be worth trying to record the stream using ffmpeg without VDH.  If you post the URL, I could concoct an ffmpeg invocation that records, let's say, 5 minutes.  You could then adapt that to be infinite, or 13 hours, & see if that works.  If that does work, then we'd have circumstantial evidence that points the finger at VDH.
